src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -861,39 +861,41 @@ description: (
 
 reference:
       (
-            PACKAGE (DOT | CLASS)* HASH? MEMBER? parameters?
-            | (DOT | CLASS)+ HASH? MEMBER? parameters?
+            PACKAGE_CLASS HASH? MEMBER? parameters?
             | HASH? MEMBER parameters?
       )
       ;
 parameters: LEFT_BRACE (ARGUMENT | COMMA | WS | NEWLINE | LEADING_ASTERISK)* RIGHT_BRACE;
 
-javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
+javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
```

Comment:
Do these changes with `WS` block current issue? If they don't then please open a new issue and move changes with `WS` in tags there.